example/generate.py

```diff
@@ -0,0 +1,71 @@
+from sys import version_info, path
+from os.path import join, abspath, dirname
+
+path.insert(0, abspath(join(dirname(__file__), "..")))
+
+from novelai_api import NovelAI_API
+from novelai_api.utils import (
+    get_encryption_key,
+    decrypt_user_data,
+    map_meta_to_stories,
+    assign_content_to_story,
+)
+from aiohttp import ClientSession
+
+from logging import Logger
+from asyncio import run
+
+import json
+
+# pip3 install pyyaml
+import yaml
+
+filename = join("credentials", "creds_example.txt")
```

Comment:
The credentials are now retrieved from environment variables (see updated examples).
This way of getting them was long overdue to leaving, as github workflow doesn't support hidden files.

Reply:
Good idea.